pymc3/gp/cov.py

```diff
@@ -207,6 +205,21 @@ def full(self, X, Xs=None):
         raise NotImplementedError
 
 
+class Periodic(Stationary):
```

Comment:
I think they can both be here.  Functions drawn from a gp w/ cosine prior are exactly sinusoidal functions.  Periodic, with its lengthscale parameter, gives functions that are periodic but not perfectly sinusoidal.